js/adapt-xapi.js

```diff
@@ -331,22 +352,44 @@ define(function(require) {
     },
 
     validateParams: function() {
-      var isValid = true;
-      if (!this.get('actor') || typeof this.get('actor') != 'object') {
-        console.log('\'actor\' is invalid');
-        isValid = false;
+      return !(
```

Comment:
can we get rid of this negation avalanche? It makes my head spin just looking at it. Break this down into a few statements, please.